pkg/operator/staticpod/controller/node/node_controller.go

```diff
@@ -67,6 +79,14 @@ func (c *NodeController) sync(ctx context.Context, syncCtx factory.SyncContext)
 		return err
 	}
 
+	if c.arbiterNodeSelector != nil {
+		arbiterNodes, err := c.nodeLister.List(c.arbiterNodeSelector)
```

Comment:
Ah, that's a good point, unfortunately, this won't work in this situation labels selectors by design do not support an `or` operation on keys ([see](https://github.com/kubernetes/kubernetes/issues/90549#issuecomment-620625847)). So this must be done as separately, otherwise if we were going of values we could build an `contains` query.

Reply:
ok, could you add a comment explaining that the OR operation is not supported on keys and include a link to the issue you mentioned in the other comment?